src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -62,38 +71,107 @@ const SearchPage = () => {
             },
           };
         });
-      setResults(finalResults);
+      setResults({ type: 'author', searchResults: finalResults });
     },
   });
 
   // Hook that will re-render the page only if the state of searchText changes
   useEffect(() => {
-    executeSearch();
-  }, [searchText]);
+    async function searchPosts() {
+      // Setting to check length > 3, as the component will trying to fetch with 0 length causing error
+      if (searchText.length > 3) {
```

Comment:
In another bug I want to add the Search button, since the way it gobbles network bandwidth to do searches while you type is no good on mobile.  But this is fine for now.

Reply:
There's a PR up for that, #839